README.md

```diff
@@ -10,3 +10,36 @@ CodeFlare Stack Compatibility Matrix
 | CodeFlare-SDK                | v0.4.4  |
 | InstaScale                   | v0.0.4  |
 | KubeRay                      | v0.5.0  |
+
+## Release process
+
+Prerequisite:
+- Build and release [MCAD](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
+- Build and release [InstaScale](https://github.com/project-codeflare/instascale)
+- Build and release [CodeFlare-SDK](https://github.com/project-codeflare/codeflare-sdk)
+
+Release steps:
+1. Invoke [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, this action will create a repository tag, build and push operator image.
+
+2. Check result of [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, it should pass.
+
+3. Update CodeFlare Stack Compatibility Matrix in operator README.
+
+4. Update InstaScale and MCAD versions:
+- in [Makefile](https://github.com/project-codeflare/codeflare-operator/blob/main/Makefile#L12-L16).
+- in [mcad/deployment.yaml.tmpl](https://github.com/project-codeflare/codeflare-operator/blob/main/config/internal/mcad/deployment.yaml.tmpl#L28)
+- in [controllers/defaults.go](https://github.com/project-codeflare/codeflare-operator/blob/main/controllers/defaults.go) by running `make defaults`
```

Comment:
I don't think that's needed, as it's done when running `make build` as part of the release workflow.

Reply:
I mention it here because the change needs to be merged, so it is good to merge it together with other version changes in one commit.